changelogs/unreleased/6095-lubronzhan-small.md

```diff
@@ -0,0 +1 @@
+Updates to Go 1.21.6. See the [Go release notes](https://go.dev/doc/devel/release#go1.21.minor) for more information.
```

Comment:
Did this file get added by accident as it is not related?  We could skip changelog entry for this change.

Reply:
Ohk let me remove that thanks

Reply:
since this one is just a tooling/CI change shouldn't need any release note, generaly guidelines are here on release notes: https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md#commit-message-and-pr-guidelines

we sorta play it by ear, but a significant new feature/behavior change should be a major note, medium added feature or bug fix minor, small change/fix -> small, etc.

Reply:
Typically we only do changelogs for user-facing changes, not developer-facing ones.